package.json

```diff
@@ -86,19 +86,20 @@
   "dependencies": {
     "@ministryofjustice/frontend": "^1.2.0",
     "agentkeepalive": "^4.2.0",
-    "applicationinsights": "^2.2.0",
+    "applicationinsights": "^2.2.1",
     "body-parser": "^1.19.1",
     "bunyan": "^1.8.15",
     "bunyan-format": "^0.2.1",
     "compression": "^1.7.4",
     "connect-flash": "^0.1.1",
-    "connect-redis": "^6.0.0",
+    "connect-redis": "^6.1.1",
```

Comment:
Wonder if this means we can tidy up the redis stuff a bit - I think it means you can use redis client with legacy mode.. will have to have a look later